modules/wmco-disconnected-cluster.adoc

```diff
@@ -4,25 +4,31 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="wmco-disconnected-cluster_{context}"]
-= Using Windows containers in a disconnected cluster
+= Using Windows containers with a mirror registry
 
-The Windows Machine Config Operator (WMCO) can be installed into a {product-title} cluster in a disconnected, or air-gapped, network. A _disconnected network_ is a restricted network without direct internet connectivity.
+The Windows Machine Config Operator (WMCO) can pull images from a registry mirror rather than from a public registry. You can pull images from the mirror registry directly or you can use you can use an `ImageDigestMirrorSet` (IDMS) or `ImageTagMirrorSet` (ITMS) object to configure your cluster to pull images from the mirror registry.
 
-Because the cluster does not have access to the internet, any required images cannot be downloaded. As such, you can create a _mirror registry_, which is a container registry within the disconnected network that contains all the container images required by {product-title} and the WMCO. 
```

Comment:
I think this info is useful. I suggest changing the first sentence here to
`Because the cluster does not have access to the internet, any external container images cannot be referenced.`

Reply:
Agreed! I moved that sentence to here: https://github.com/openshift/openshift-docs/pull/74918/files#diff-038a6106c09d0b2f6ac1faf140b33c1091d367447b46c12fd8384da289646369R17 And updated at your suggestion.